types/expo/index.d.ts

```diff
@@ -1855,19 +1856,36 @@ export namespace Notifications {
         };
     }
 
+    interface SchedulingOptions {
+        time: Date | number;
+        repeat?: 'minute' | 'hour' | 'day' | 'week' | 'month' | 'year';
+        intervalMs?: number;
+    }
+
+    interface ChannelAndroid {
+        name: string;
+        description?: string;
+        sound?: boolean;
+        priority?: 'min' | 'low' | 'default' | 'high' | 'max';
+        vibrate?: boolean | number[];
+        badge?: boolean;
+    }
+
     type LocalNotificationId = string | number;
 
     function addListener(listener: (notification: Notification) => any): EventSubscription;
     function getExpoPushTokenAsync(): Promise<string>;
     function presentLocalNotificationAsync(localNotification: LocalNotification): Promise<LocalNotificationId>;
     function scheduleLocalNotificationAsync(
         localNotification: LocalNotification,
-        schedulingOptions: { time: Date | number, repeat?: 'minute' | 'hour' | 'day' | 'week' | 'month' | 'year' }
+        schedulingOptions: SchedulingOptions
     ): Promise<LocalNotificationId>;
     function dismissNotificationAsync(localNotificationId: LocalNotificationId): Promise<void>;
     function dismissAllNotificationsAsync(): Promise<void>;
     function cancelScheduledNotificationAsync(localNotificationId: LocalNotificationId): Promise<void>;
     function cancelAllScheduledNotificationsAsync(): Promise<void>;
+    function createChannelAndroidAsync(id: string, channel: ChannelAndroid): Promise<void>;
```

Comment:
I think it would be nice to include the documentation from expo here in the file, as it has been done in other areas in here.

Reply:
Just to make sure I understand you correctly: what you meant is to add the docs in a comment (`/** ... */`), right?

Reply:
I don't see any example of documentation pasted from expo.io. And for this function itself, there is a LOT of text to be copied and pasted... Not sure if that's what we want

Reply:
@janaagaard75 do you mean for this specific method or above the whole section?

Reply:
Yes, I meant copying the docs from docs.expo.io into `/** ... */ comments and/or writing a better comments where we can. I can see that this has only been done in a few places - the `Calendar` is one example.

And yes, I think this should be done for the whole section.

I should have stressed that this is definitely a nice to have. I you don't have the time to implement this, @burtek, or don't think that we should do this, then don't bother. Having correct typedefs is much more important.

Reply:
Maybe next time, it would be a little too much time consuming now.